cms/jinja2/templates/pages/methodology_page.html

```diff
@@ -10,43 +10,52 @@
 {%- set breadcrumbs_label = _("Breadcrumbs") -%}
 
 {% block header_area %}
-    {% if page.contact_details %}
-        {% set contact_description_html -%}
-            <a href="mailto:{{ page.contact_details.email }}">{{ page.contact_details.name }}</a>
-        {%- endset %}
-
-        {# fmt:off #}
-        {% set contact_details =
+    {# fmt:off #}
+    {% set publication_item = {
+        "term": published_label,
+        "descriptions": [
             {
-                "term": "Contact:",
+                "description": page.publication_date|date("DATE_FORMAT")
+            }
+        ]
+    } %}
+    {# fmt:on #}
+
+    {# fmt:off #}
+    {% if page.last_revised_date %}
+        {% set last_revised_item = {
+                "term": last_revised_label,
                 "descriptions": [
                     {
-                        "description": contact_description_html
+                        "description": page.last_revised_date|date("DATE_FORMAT")
                     }
                 ]
             }
         %}
-        {# fmt:on #}
     {% else %}
-        {% set contact_details = "" %}
+        {% set last_revised_item = None %}
     {% endif %}
+    {# fmt:on #}
 
-    {% if page.last_revised_date %}
-        {# fmt:off #}
-        {% set last_revised =
-            {
-                "term": last_revised_label,
+    {% set contact_description_html -%}
+        <a href="mailto:{{ page.contact_details.email }}">{{ page.contact_details.name }}</a>
+    {%- endset %}
+
+    {# fmt:off #}
+    {% if page.contact_details %}
+        {% set contact_item = {
+                "term": _("Contact:"),
```

Comment:
I think we might need to move this to a variable like we did before so that it can be localised (here and in other places where we use an object/dictionary)

Reply:
Good shout thank you - I've moved these to the top with the other labels now 👍